.travis.yml

```diff
@@ -1,11 +1,10 @@
 # Syntax-check Travis YAML .travis.yml files at http://lint.travis-ci.org
 # Force Travis to rebuild by commiting an increment to this counter: 5
 
-language: c
+language: cpp
```

Comment:
@czender, this makes the travis project a C++ project.

Previously, there were 3 jobs.
- export CC=gcc       (CXX=g++ implicitly)
- export CC=clang    (CXX=g++ implicitly - This does not work well, hence the travis failures in master)
- export CC=g++      (CXX=g++ implicitly - This is a weird combination, is there a need for this?)

Now, there are 2 jobs
- export CC=gcc && export CXX=g++
- export CC=clang && export CXX=clang++

So, the C files are compiled with `${CC}` and the C++ files are compiled with `${CXX}`

Reply:
So, in other words, you have fixed the Travis failure (that first appeared in with the Ubuntu upgrade to Trusty in ~October) by switching from implicit to explicit CXX=clang++ with the clang build. What has been lost is testing the C99 code with g++. Can you make a target that uses CC=g++ and CXX=g++? and possibly one with CC=clang++ and CXX=clang++? Those would catch C++-incompatibilities that were inadvertently introduced into the C99 code. It helps to know this because the Windows build must use a C++ compiler for all the code (because MSVC has no C99 compiler).